packages/block-editor/src/components/block-list/block-list-context.native.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext, useContext } from '@wordpress/element';
+
+export const DEFAULT_BLOCK_LIST_CONTEXT = {
+	scrollRef: null,
+	blocksLayouts: { current: {} },
+	updateBlocksLayouts,
+};
+
+const Context = createContext( DEFAULT_BLOCK_LIST_CONTEXT );
+const { Provider, Consumer } = Context;
+
+/**
+ * Finds a block's layout data within the provided data object.
+ *
+ * @param {Object} data     Blocks layouts list.
```

Comment:
Nitpick: Probably it's not accurate to use the word `list` here, as the blocks layouts data is an object.
```suggestion
 * @param {Object} data     Blocks layouts object.
```

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/39089/commits/b1ee8f60d54f96f3c5e723b1de903928357122da